### PR TITLE
Gdr 1936

### PR DESCRIPTION
**Pull request by DominikZabinski**
# Description
## What changed?
Related JIRA issue: 
GDR-1936
## Why was it changed?
Removing redundant files
# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [x] Package version bumped
- [ ] Changelog updated

# Screenshots (optional)

### CODE REVIEW

**Review by gladkia — APPROVED** [review]

File: inst/testdata/synthetic_list.yml [hunk 1]
```diff
@@ -6,9 +6,6 @@
 - name: finalSE_combo_2dose_nonoise
   file: finalSE_combo_2dose_nonoise.RDS
   description: 
-- name: finalSE_many_drugs
-  file: finalSE_many_drugs.RDS
-  description:
```
Comment (j-smola) [inline inst/testdata/synthetic_list.yml:11]: Shouldn't `finalSE_combo_1dose_many_drugs` and `finalSE_many_lines` also be removed - as the task description listed?

**Review by j-smola — COMMENTED** [review]
1. Task description lists three datasets to be deleted. 
- finalMAE_many_lines.RDS
- finalMAE_many_drugs.RDS
- finalMAE_combo_1dose_many_drugs.RDS 
The first two are deleted, but the third remains. Has there been a change in the scope of tasks? 

2. Why synthetic_list.RDS was removed?

File: inst/testdata/synthetic_list.yml (hunk 1 above)
Comment (DominikZabinski) [reply inst/testdata/synthetic_list.yml:11]: Updated task description, thank you.

**Comment by DominikZabinski** [comment]
> 1. Task description lists three datasets to be deleted.
> 
> * finalMAE_many_lines.RDS
> * finalMAE_many_drugs.RDS
> * finalMAE_combo_1dose_many_drugs.RDS
>   The first two are deleted, but the third remains. Has there been a change in the scope of tasks?
> 
> 2. Why synthetic_list.RDS was removed?

1. My mistake during resolving conflicts.
2. Updated task description.

File: inst/testdata/synthetic_list.yml (hunk 1 above)
Comment (j-smola) [reply inst/testdata/synthetic_list.yml:11]: `synthetic_list.yml` should be also updated - please, remove also description for `finalSE_combo_1dose_many_drugs` and `finalSE_many_lines` from this list.

**Review by j-smola — APPROVED** [review]

File: inst/testdata/synthetic_list.yml (hunk 1 above)
Comment (DominikZabinski) [reply inst/testdata/synthetic_list.yml:11]: As requested, updated task descritpion.

**Pull request merged**